rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] **[OPTIONAL]** The `book` and `person` properties should have `attr_accessor`.